setup.cfg

```diff
@@ -129,6 +129,4 @@ python_files =
     test_*.py
```

Comment:
Note about your commit message: The command to create a demo app could move to the sandbox... it is not really part of the app itself.

Reply:
Unless it is distributed with the application itself?

Reply:
If you think factories must be part of the tests only and that this management command should stand in the sandbox, let's do it! (in a new PR)

Reply:
FTR it has been decided that we will distribute factories and the create demo app management command. That can be useful when working with Richie in a new Django-CMS project.